csharp/OnnxRuntime.CSharp.proj

```diff
@@ -61,12 +70,48 @@ CMake creates a target to this project
     </Exec>
   </Target>
 
+   <Target Name="ObtainPackageVersion" BeforeTargets="CreatePackage">
+    <ReadLinesFromFile File="..\VERSION_NUMBER">
+      <Output TaskParameter="Lines" ItemName="MajorVersionNumber"/>
+    </ReadLinesFromFile>   
+    <Exec Command="git rev-parse --short HEAD" ConsoleToMSBuild="true">
+        <Output TaskParameter="ConsoleOutput" PropertyName="GitCommitHashShort" />
+    </Exec>
+    <Exec Command="git rev-parse HEAD" ConsoleToMSBuild="true">
```

Comment:
Need this extra command in order for us to add some metadata in the native Nuget to match the current metadata available in the release nugets.